integration_test/open_group_test.js

```diff
@@ -23,25 +23,25 @@ describe('Open groups', function() {
     await common.killallElectron();
   });
 
-  it('works with valid group url', async () => {
+  // reduce code duplication to get the initial join
+  async function joinOpenGroup(url, name) {
```

Comment:
could be in `common.js`

Reply:
no, I feel like common shouldn't have functions for specific tests. In fact I'd favor moving some of the constants out of it. I think we should favor smaller cleaner files (even though, I personally have no problem working with bigger ones).